posthog/exceptions.py

```diff
@@ -45,7 +45,7 @@ class Conflict(APIException):
 
 class ClickhouseAtCapacity(APIException):
     status_code = 500
-    default_detail = "Clickhouse cluster is at capacity. Please try this query again later."
+    default_detail = "We are a little busy right now. Please try this query again later."
```

Comment:
```suggestion
    default_detail = "Queries are a little too busy right now. We're working to free up resources. Please try again later."
```

Maybe? So it doesn't feel like "posthog gets busy and then just dies"